Sortable.js

```diff
@@ -215,7 +215,9 @@
 			delay: 0,
 			forceFallback: false,
 			fallbackClass: 'sortable-fallback',
-			fallbackOnBody: false
+			fallbackOnBody: false,
+			xOffset: 0,
```

Comment:
Please, replace `xOffset` and `yOffset` -> `fallbackOffset: {x: 0, y: 0}`
And also forward PR into `dev`-branch.